server/app/services/applicant/ApplicantService.java

```diff
@@ -1153,15 +1158,17 @@ private ApplicationPrograms relevantProgramsForApplicantInternal(
             }
             programNamesWithApplications.add(programName);
           } else if (maybeSubmittedApp.isPresent() && activeProgramNames.containsKey(programName)) {
-            // When extracting the application status, the definitions associated with the program
-            // version at the time of submission are used. However, when clicking "reapply", we use
-            // the latest program version below.
+            // When extracting the application status, the Statuses present in the current program
+            // versions are used
```

Comment:
Since we renamed it, I will update the comment. Or in the other hand, we are calling it activeStatusDefinitions now, I don't see the need for this comment.